src/components/Calendar/doubleCalendar/hooks/useDisabledControls.js

```diff
@@ -0,0 +1,22 @@
+import { useMemo } from 'react';
+import { addMonths } from '../../helpers';
+import useLastDayMonth from './useLastDayMonth';
+
+export default function useDisabledControls(
+    yearsRange,
+    minDate,
+    maxDate,
+    currentMonth,
+    rightCalendarMonth,
+) {
+    const lastDayMonth = useLastDayMonth(currentMonth);
```

Comment:
const lastDayMonth = getLastDayMonth(addMonths(currentMonth, -1)) inside the useMemo and you don't need this hook anymore.